### PR TITLE
Improved The Documentation

### PR DESCRIPTION
**Pull request by NPDebs**
This PR addresses issue #11.
A detailed documentation has been added, including:

- a detailed overview of the theme, explaining its purpose and features.
- step-by-step instructions on how to install the theme in a Jekyll project.
- information on how users can seek support, report issues, and give feedback for continuous improvement.

Thanks for reviewing!

### CODE REVIEW

**Comment by NPDebs** [comment]
@bebatut Here's the documentation. Please review when you can. 🙏🏻

**Comment by NPDebs** [comment]
@bebatut Here's the documentation. Please review when you can. 🙏🏻

**Review by bebatut — COMMENTED** [review]
Thanks @NPDebs

File: README.md [hunk 1]
```diff
@@ -27,24 +28,52 @@ And then execute:
...
+If you encounter any issues or have suggestions for improvements, please feel free to reach out to us at [team@we-are-ols.org](team@we-are-ols.org). We value your feedback and are here to assist you.
+
+Thank you!
 
 ## License
 
-The theme is available as open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
+The content of this website are licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
+
+The code behind the infrastructure is licensed under the [MIT License](LICENSE.txt).
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply README.md:79]: Fair point. 😃 I'll remove that part now...

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-To experiment with this code, add some sample content and run `bundle exec jekyll serve` – this directory is setup just like a Jekyll site!
+## Getting Started
 
-TODO: Delete this and the text above, and describe your gem
+To use `ols-theme` in your Jekyll project, follow these steps:
 
-## Installation
+### Installation
 
 Add this line to your Jekyll site's `Gemfile`:
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply README.md]: Yes, please.

File: README.md [hunk 3]
```diff
@@ -27,24 +28,52 @@ And then execute:
...
 
-TODO: Write usage instructions here. Describe your available layouts, includes, sass and/or assets.
+To see if the installation has been done correctly, run `bundle exec jekyll serve` and open your browser at `http://localhost:4000`. This starts a Jekyll server to enable you preview the site.
 
-## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/ols-theme. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](https://www.contributor-covenant.org/) code of conduct.
+## Theme Structure
+- *_includes*:
+ Includes reusable components used within the layouts.
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply README.md]: @betat..

[the remark this answers is not in the document]
Comment (bebatut) [reply README.md]: No. I meant use the same formatting with folder name between "`" and then the description on the same line

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (NPDebs) [reply README.md]: Done. Thanks a lot!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (NPDebs) [reply README.md]: Done. 👍

File: Update description of theme files [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+2. Add this line to your Jekyll site's `_config.yml`:
+
+```yaml
+theme: ols-theme
+```
+
+3. Execute:
+
+   $ bundle
```
Comment (bebatut) [inline Update description of theme files:26]: Can you put "```" before and after the commands? Thanks

File: Update description of theme files [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+- `Gemfile`: File specifying Ruby dependencies for the theme.
+
+- `Gemfile.lock`: Lock file ensuring consistent installation of gems.
+
+- `README.md`: Instructions and information about the theme.
+
+- `code-of-conduct.md`: Community code of conduct.
+
+- `ols-theme.gemspec`: Gem specification file for RubyGems packaging.
+
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply Update description of theme files]: I followed the same pattern as is in the main OLS repo => [COC 1](https://github.com/open-life-science/open-life-science.github.io/blob/main/CODE_OF_CONDUCT.md) and [COC 2](https://github.com/open-life-science/open-life-science.github.io/blob/main/code-of-conduct.md).

File: Update description of theme files (hunk 4 above)
Comment (NPDebs) [reply Update description of theme files]: Yes, please. Done!

**Pull request merged**